release/7-1/arm32v7/meta.json

```diff
@@ -7,12 +7,15 @@
     "Base64EncodePackageUrl": true,
     "UseAcr": true,
     "tagTemplates": [
-        "#psversion#-arm32v7-ubuntu-bionic-#fulltag#",
-        "#psversion#-arm32v7-ubuntu-18.04-#fulltag#",
-        "arm32v7-ubuntu-#shorttag#"
+        "#psversion#-ubuntu-#shorttag#-arm32-#fulltag#",
+        "ubuntu-#shorttag#-arm32"
     ],
     "shortTags": [
-        {"Tag": "bionic"}
+        {"Tag": "bionic"},
+        {"Tag": "18.04"}
+    ],
+    "manifestLists": [
+        "7-1"
```

Comment:
not sure if this should be "7-1" or not be in any manifest list. Also on L11 does that image name become `7-1-ubuntu-#shorttag#-arm32`?

Reply:
7-1 I think is right.